lhotse/cut.py

```diff
@@ -1152,6 +1155,80 @@ def truncate(
             supervisions=sorted(supervisions, key=lambda s: s.start),
         )
 
+    def extend(
+        self,
+        *,
+        duration: Seconds,
+        direction: str = "both",
+        preserve_id: bool = False,
+    ) -> "MonoCut":
+        """
+        Returns a new MonoCut that is an extended region of the current MonoCut.
+
+        Note that no operation is done on the actual features or recording -
+        it's only during the call to :meth:`MonoCut.load_features` / :meth:`MonoCut.load_audio`
+        when the actual changes happen (an extended version of features/audio is loaded).
+
+        .. hint::
+
+            This method extends a cut by a given duration, either to the left or to the right (or both).
+            To "expand" a cut by padding, use :meth:`MonoCut.pad`. To "truncate" a cut, use :meth:`MonoCut.truncate`.
+
+        :param duration: float (seconds), controls the duration of the resulting MonoCut.
+        :param direction: string, 'left', 'right' or 'both'. Determines whether to extend on the left,
+            right, or both sides. If 'both', extend on both sides by the duration specified in `duration`.
```

Comment:
I think this is inconsistent with other methods like `pad` which, when direction="both", would extend in each direction by `duration/2`

Reply:
Should I change it to be consistent with the `pad` usage?

Reply:
After the discussion I think it makes sense to leave it the way you did, just make sure it's explicit in the documentation